qa/tasks/persistent_write_log_cache_thrash.py

```diff
@@ -0,0 +1,99 @@
+"""
+persistent write log cache thrash task
+"""
+import contextlib
+import logging
+import random
+
+from teuthology import misc as teuthology
+from teuthology import contextutil
+from teuthology.orchestra import run
+
+DEFAULT_KILL_DAEMON_TIME = 20
```

Comment:
`rbd bench` is not a daemon:
```suggestion
DEFAULT_NUM_ITERATIONS = 20
```